azext_edge/edge/providers/check/base/resource.py

```diff
@@ -468,3 +470,30 @@ def process_custom_resource_status(
                     namespace=namespace,
                     display=Padding(status_text, (0, 0, 0, padding + 4)),
                 )
+
+
+def validate_ref(name: str, namespace: str, ref_type: ValidationResourceType) -> Tuple[str, bool]:
+    ref_obj = None
+    if ref_type == ValidationResourceType.secret:
+        ref_obj = get_namespaced_secret(secret_name=name, namespace=namespace)
+    elif ref_type == ValidationResourceType.configmap:
+        ref_obj = get_config_map(name=name, namespace=namespace)
+
+    return bool(ref_obj)
+
+
+def get_valid_references(
+    api: EdgeResourceApi, kind: Union[Enum, str], namespace: Optional[str] = None
+) -> Dict[str, Any]:
```

Comment:
From the implementation it looks like this returns a `Dict[str, bool]` - if the `key` is what we're really looking for and the `value` is unused, perhaps we could just return a `list[str]` that contains the valid reference names?

something like:

```
if custom_objects:
    return [obj.get('metadata', {}).get('name') for obj in custom_objects]
```

I think we also have a helper class somewhere to just return the `metadata.name` of a resource

Reply:
thanks for the catch, looks like I removed the usage of "value" but forgot to update the function😢